Pages/About.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
Just a note, traditionally in the root folder of an HTML project there's a file named `index.html` which is the homepage for the site.  You're missing that.